src/utils/formatNumbers.ts

```diff
@@ -510,50 +526,15 @@ export const priceToPreciseFloat = (price: Price<Currency, Currency> | undefined
   return floatForLargerNumbers
 }
 
-/**
- * Returns a numerical amount of any token formatted in human readable string for use in template.
- *
- * For transaction review numbers, such as token quantities, NFT price (token-denominated),
- *  network fees, transaction history items. Adheres to guidelines defined here:
- * https://www.notion.so/uniswaplabs/Number-standards-fbb9f533f10e4e22820722c2f66d23c0
- * @param num numerical value denominated in any token
- * @param maxDigits the maximum number of digits that should be shown for the quantity
- */
-export const formatTransactionAmount = (num: number | undefined | null, maxDigits = 9) => {
-  if (num === 0) return '0.00'
-  if (!num) return ''
-  if (num < 0.00001) {
-    return '<0.00001'
-  }
-  if (num >= 0.00001 && num < 1) {
-    return `${Number(num.toFixed(5)).toLocaleString(DEFAULT_LOCALE, {
-      minimumFractionDigits: 2,
-      maximumFractionDigits: 5,
-    })}`
-  }
-  if (num >= 1 && num < 10000) {
-    return `${Number(num.toPrecision(6)).toLocaleString(DEFAULT_LOCALE, {
-      minimumFractionDigits: 2,
-      maximumFractionDigits: 6,
-    })}`
-  }
-  if (num >= 10000 && num < 1000000) {
-    return `${Number(num.toFixed(2)).toLocaleString(DEFAULT_LOCALE, { minimumFractionDigits: 2 })}`
-  }
-  // For very large numbers, switch to scientific notation and show as much precision
-  // as permissible by maxDigits param.
-  if (num >= Math.pow(10, maxDigits - 1)) {
-    return `${num.toExponential(maxDigits - 3)}`
-  }
-  return `${Number(num.toFixed(2)).toLocaleString(DEFAULT_LOCALE, { minimumFractionDigits: 2 })}`
-}
-
 const MAX_AMOUNT_STR_LENGTH = 9
 
-export function formatReviewSwapCurrencyAmount(amount: CurrencyAmount<Currency>): string {
-  let formattedAmount = formatCurrencyAmount({ amount, type: NumberType.TokenTx })
+export function formatReviewSwapCurrencyAmount(
+  amount: CurrencyAmount<Currency>,
+  locale: SupportedLocale = DEFAULT_LOCALE
+): string {
```

Comment:
to avoid confusion i think you should have one entrypoint/interface to the whole suite of formatters. it seems like the hook below is the interface. so we shouldn't export these functions anymore.

if there's a use case for formatting these values with locales that are not the "active" `formatterLocale` then you should build that into the interface below. 

i just think having the interface surface be this wide/unconstrained will make it easy for new users of formatters to use the wrong function and introduce an incorrectly formatted value in the future.

Reply:
agree with this - making this a draft to refactor it to this

Reply:
actually the reason for this is for the tests - in a pr ontop of this i am going to do this refactor

Reply:
this was done in this pr: #7308